internal/hcsoci/resources_wcow.go

```diff
@@ -147,6 +148,13 @@ func setupMounts(ctx context.Context, coi *createOptionsInternal, r *resources.R
 					return errors.Wrapf(err, "adding SCSI virtual disk mount %+v", mount)
 				}
 				r.Add(scsiMount)
+			} else if mount.Type == "extensible-virtual-disk" {
+				l.Debug("hcsshim::allocateWindowsResource Hot-adding ExtensibleVirtualDisk")
```

Comment:
We just had a round of log cleanup where we swapped to the package name for the prefix before :: to make it more clear where it's coming from. The rest of the mounts all have their logs as this format though so likely better for a different pr

Reply:
I would change it but it will have to be changed for all the files in this package. So it would be better to do this in a separate PR.